cpp/doAll_Zp.C

```diff
@@ -261,15 +274,17 @@
 	if (year.Contains("2016") && sample=="DYbb")
 	  sample_names[sample] = "DYBBJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-pythia8";
         TString dir = baseDir+"/"+sample_names[sample]+"_"+sample_prod[sample][year][d]+"_"+dataformat+"_"+skimPackage+"/merged/merged.root";
-        if (runOnSignalBeforeSkim)
+        if ( sample.Contains("BFF") && !(sample.Contains("BFF_M250")) )
```

Comment:
Why the difference for M250 sample?

Reply:
This is because as of today the path for M=250 GeV is different from the others.
Can be removed once the paths are homogeneous, though:
- BFFZprimeToMuMu_M_250_TuneCUETP8M1_13TeV-madgraph-pythia8_RunIISummer16NanoAODv7-PUMoriond17_Nano02Apr2020_102X_mcRun2_asymptotic_v8-v1_NANOAODSIM**_skim2mu_1muPt50_1Mll100_allBranches_allFiles**/
- BFFZprimeToMuMu_M_350_TuneCUETP8M1_13TeV-madgraph-pythia8_RunIISummer16NanoAODv7-PUMoriond17_Nano02Apr2020_102X_mcRun2_asymptotic_v8-v1_NANOAODSIM/
- BFFZprimeToMuMu_M_350_dbs1p0_TuneCUETP8M1_13TeV-madgraph-pythia8_RunIISummer16NanoAODv7-PUMoriond17_Nano02Apr2020_102X_mcRun2_asymptotic_v8-v1_NANOAODSIM/
- BFFZprimeToMuMu_M_500_TuneCUETP8M1_13TeV-madgraph-pythia8_RunIISummer16NanoAODv7-PUMoriond17_Nano02Apr2020_102X_mcRun2_asymptotic_v8-v1_NANOAODSIM/

Reply:
Sorry, my mistake when naming the directories. Fixed:
```
[evourlio@uaf-10.../evourlio/skimOutput/skim2mu_1muPt50_1Mll100_allBranches_allFiles]$ ls BFFZprimeToMuMu_M_ 
BFFZprimeToMuMu_M_250_TuneCUETP8M1_13TeV-madgraph-pythia8_RunIISummer16NanoAODv7-PUMoriond17_Nano02Apr2020_102X_mcRun2_asymptotic_v8-v1_NANOAODSIM/
BFFZprimeToMuMu_M_350_TuneCUETP8M1_13TeV-madgraph-pythia8_RunIISummer16NanoAODv7-PUMoriond17_Nano02Apr2020_102X_mcRun2_asymptotic_v8-v1_NANOAODSIM/
BFFZprimeToMuMu_M_350_dbs1p0_TuneCUETP8M1_13TeV-madgraph-pythia8_RunIISummer16NanoAODv7-PUMoriond17_Nano02Apr2020_102X_mcRun2_asymptotic_v8-v1_NANOAODSIM/
BFFZprimeToMuMu_M_500_TuneCUETP8M1_13TeV-madgraph-pythia8_RunIISummer16NanoAODv7-PUMoriond17_Nano02Apr2020_102X_mcRun2_asymptotic_v8-v1_NANOAODSIM/
```

The M250 exception can be removed, thanks!

Reply:
Removed, thank you!